apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-steps/plan-step/index.tsx

```diff
@@ -20,20 +20,18 @@ const PlanStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep, onN
 	return (
 		<LaunchStep className="nux-launch-plan-step">
 			<div className="nux-launch-step__header">
-				<div className="nux-launch-step__heading">
-					<h1 className="nux-launch-step__title">{ __( 'Choose a plan', 'full-site-editing' ) }</h1>
-					<p className="nux-launch-step__subtitle">
+				<div>
+					<Title>{ __( 'Choose a plan', 'full-site-editing' ) }</Title>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.